srcpkgs/Ice/template

```diff
@@ -1,86 +1,60 @@
 # Template file for 'Ice'
 pkgname=Ice
-version=3.5.1
-revision=22
-build_wrksrc=cpp
-makedepends="zlib-devel bzip2-devel mcpp-devel db-devel expat-devel libressl-devel"
+version=3.7.5
+revision=1
+wrksrc="ice-${version}"
+build_style=gnu-makefile
+make_build_args="LANGUAGES=cpp prefix=${DESTDIR}/usr OPTIMIZE=yes
+ USR_DIR_INSTALL=yes CONFIGS=shared CONFIGS+=cpp11-shared"
+make_install_args="LANGUAGES=cpp prefix=${DESTDIR}/usr"
+makedepends="zlib-devel bzip2-devel zeroc-mcpp db-devel expat-devel
+ libressl-devel lmdb-devel editline-devel"
 short_desc="Internet Communications Engine (Ice)"
-maintainer="Orphaned <orphan@voidlinux.org>"
-license="GPL-2.0-only, ICE"
-homepage="http://www.zeroc.com"
-distfiles="${homepage}/download/Ice/${version%.*}/${pkgname}-${version}.tar.gz"
-checksum=989e51194c6adadbd156da3288e37bad847b93b3b876502e83033b70493af392
-python_version=2 #unverified
+maintainer="a dinosaur <nick@a-dinosaur.com>"
+license="GPL-2.0-only, custom:ICE"
+homepage="https://www.zeroc.com"
+distfiles="https://github.com/zeroc-ice/ice/archive/v${version}.tar.gz"
+checksum=36bf45591a95e6ee7216153d45d8eca05ff00c1da35608f0c400e6ddc8049da9
+patch_args="-Np1"
 
 if [ "$CROSS_BUILD" ]; then
-	# Cross build requires the host's slice2cpp and slice2freeze
+	# Cross build requires the host's slice2cpp
 	hostmakedepends+=" Ice"
 fi
 
 do_configure() {
-	sed -i -e "s|^#OPTIMIZE|OPTIMIZE|" config/Make.rules
-	sed -i -e "s|x)/config|x)/lib/Ice/config|" config/Make.rules
-	sed -i '/^embedded_runpath_prefix*/d' config/Make.rules
-	sed -i -e 's|x)/man/man1|x)/share/man/man1|' config/Make.rules
-	sed -i -e "s|test demo||" Makefile
+	# Disable building tests
+	vsed 's|include \$(top_srcdir)/config/Make.tests.rules||' -i config/Make.rules
 
 	# Simple fixes for musl libc instead of a proper patch
-	sed -i -e 's|__GNUC__|__GLIBC__|' src/IceUtil/Exception.cpp
-	sed -i -e 's|PTHREAD_MUTEX_RECURSIVE_NP|PTHREAD_MUTEX_RECURSIVE|' src/IceUtil/RecMutex.cpp
-	sed -i -e 's|sys/poll\.h|poll.h|' src/Ice/*.h
+	vsed 's|sys/poll\.h|poll.h|' -i cpp/src/Ice/Network.h cpp/src/Ice/Selector.h
```

Comment:
I don't think we need this, just let it print the warning.

Reply:
Holdover from the old template, didn't realise it only printed a warning.